webring/README.md

```diff
@@ -0,0 +1,116 @@
+# Webring Plugin
+
+This plugin retrieves the latest posts from a list of web feeds and makes
+them available in templates, effectively creating a [partial webring]. Posts
```

Comment:
I think this was supposed to be a link, which presumably means it should instead be: `[partial webring][]`

Reply:
In fact it's a [reference link], specifically a `link label` :open_mouth: 

[reference link]: https://github.github.com/gfm/#reference-link "This is a reference link, too!"

Reply:
Hmm. Not familiar with that bit of (presumably) GitHub-specific Markdown syntax. Not sure I see the benefit of deviating from standard Markdown just to save two characters, but perhaps I'm misunderstanding something. 🤷‍♂️

Reply:
I was the one misunderstanding something :disappointed: